composer.patches.json

```diff
@@ -30,6 +30,9 @@
         },
         "mailchimp/marketing": {
             "PHP 8.2": "PATCHES/mailchimp_php82.patch"
+        },
+        "unocha/common_design": {
```

Comment:
This has been merged already, do we want a new release now? Or to test it here first and remove the patch later.

Reply:
Let's wait till next week, but i'm going to merge this so it can be tested on dev